src/atomic.dex.mm2.cpp

```diff
@@ -617,12 +623,12 @@ namespace atomic_dex
             spdlog::error("{}", ec.message());
         }
 
-        m_mm2_init_thread = std::thread([this]() {
+        m_mm2_init_thread = std::thread([this, mm2_cfg_path]() {
             using namespace std::chrono_literals;
             // loguru::set_thread_name("mm2 init thread");
 
             const auto wait_ec = m_mm2_instance.wait(2s).second;
-
+            fs::remove(mm2_cfg_path);
```

Comment:
Here @artemii235 I highly recommend from MM2 core to delete the file themselves once you have parsed it, It's will be much more faster than me, i have no idea when you don't need the file anymore

Reply:
We can pass through another env variable if you prefer:

`MM_TEMP_CONF_FILE` (if you want to keep the other behavior)